units/include/scipp/units/simple.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+/// @file Simple and small system of units for testing purposes.
+/// @author Simon Heybrock
+#ifndef SCIPP_UNITS_SIMPLE_H
+#define SCIPP_UNITS_SIMPLE_H
+
+#include "scipp/units/unit_impl.h"
+
+namespace scipp::units {
+#ifdef SCIPP_UNITS_SIMPLE
+inline
+#endif
+    namespace simple {
+using supported_units = decltype(detail::make_unit(
+    std::make_tuple(m), std::make_tuple(dimensionless, dimensionless / m, s,
```

Comment:
how about adding also `kg` to make it a standard MKS system?

Reply:
I think I will rename this one to something unit-testing related. We can add a minimum "usuable" system later.